terraform/environments/sprinkler/platform_backend.tf

```diff
@@ -8,7 +8,7 @@ terraform {
     encrypt              = true
     key                  = "terraform.tfstate"
     region               = "eu-west-2"
-    workspace_key_prefix = "environments/accounts/sprinkler" # This will store the object as environments/sprinkler/${workspace}/terraform.tfstate
+    workspace_key_prefix = "environments/accounts/sprinkler" # This will store the object as environments/accounts/sprinkler/${workspace}/terraform.tfstate
```

Comment:
I notice this will change the location of the state file, but looking in the S3 bucket it's already in this location and last accessed in November, so Im not sure if it was changed accidentally to it's current location, but either way this should be fine.  You may want to delete the file in the old location to tidy up.

Reply:
oh yes, well noticed, weird, but this looks like the correct path, so yes, I will delete the other state and keep this one;
thanks Dave!